subjects/road_intersection/audit/README.md

```diff
@@ -1,86 +1,98 @@
 
-##### Try to generate five vehicles using the `"Up"` key, at the same time generate two vehicles using the key `"Right"`.
+##### Try to generate five vehicles using the `"Up"` key, at the same time generate two vehicles using the `"Right"` key.
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate one vehicle for all the lanes (to this at least 3 times).
+##### Try to generate one vehicle for all the lanes (do this at least 3 times).
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
 ##### Try to generate vehicles randomly using the `"R"` key. Then wait for at least 1 min.
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-###### Was there a low traffic congestion while running the application? (ex: high traffic congestion can be 8 or more vehicles in the same lane without proceeding)?
+##### Try to generate multiple vehicles with lanes at your choice.
+
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate vehicles with lanes at your choice.
+##### Try pressing the `"Esc"` key.
 
-###### Did all vehicles passed the intersection without any collision?
+###### Was the simulation finished?
+
+###### Was there a low traffic congestion while running the application? (ex: high traffic congestion can be 8 or more vehicles in the same lane without proceeding)?
 
 #### General
 
-###### Is it not possible to spam the creation of vehicles(by pressing the arrow keys to many times)?
+###### Can you confirm that it is impossible to spam the creation of vehicles (by pressing the arrow keys to many times or leave it pressed)?
 
-###### Are vehicles assigned to their own route, and if so do they obey that route? (you can use the colors for each route to see this)
+###### Are vehicles assigned to their own route, and if so, do they obey that route? (you can use the colors for each route to see this, ask the owner of the raid for instructions)
 
-###### Are vehicles kept in a safe distance(do not collide when ever one stops)?
+###### Do the vehicles kept a safe distance (do not collide whenever one stops)?
 
-###### Do vehicles stop when ever there is a red light?
+###### Do vehicles stop whenever there is a red light?
 
-###### Do vehicles proceed when ever there is a green light?
+###### Do vehicles proceed whenever there is a green light?
 
 #### Bonus
 
-###### +Is there any type of image sprite for traffic light?
+###### +Is there any type of image sprite for traffic light? (An image sprite is a collection of images put into a single image.)
 
 ###### +Did the student implement some kind of animation and image sprite for the vehicle?
 
```

Comment:
We don't seem to check if the project implements a 4-lane intersection.

Reply:
There is no need for it as we ask from line 7 to 19 if the prog creates a vehicle in each of the lanes

Reply:
My comment is irrelevant. We're using single lane carriageways.